### PR TITLE
[IMPL] NoBob

### PR DESCRIPTION
**Pull request by Wolfsurge**
Lets you use view bobbing without 'bobbing' the crosshair, e.g. tracers bobbing etc.

### CODE REVIEW

**Comment by Sxmurai** [comment]
EntityRenderer#orientCamera() does that fine when using tracers, but this works too. also, im going to make a review cause linus is gonna want some changes

**Comment by Wolfsurge** [comment]
yeah ok

**Comment by Wolfsurge** [comment]
Fuck I need to add a space after if

**Comment by notperry12345678902** [comment]
whats a norender

**Comment by master7720** [comment]
> whats a norender

xD

**Comment by master7720** [comment]
Dude just make it a NoRender setting

**Comment by Wolfsurge** [comment]
> Dude just make it a NoRender setting

fine

File: src/main/java/cope/cosmos/client/features/modules/visual/NoRender.java [hunk 1]
```diff
@@ -218,4 +220,13 @@ public void onFOVModifier(ModifyFOVEvent event) {
             event.setCanceled(true);
         }
     }
+
+    @SubscribeEvent
+    public void onCrosshairBob(CrosshairBobEvent event) {
+        // Lets the hand bobbing animation run without the crosshair bobbing as well
+        if(crosshairBob.getValue()) {
```
Comment (linustouchtips) [inline src/main/java/cope/cosmos/client/features/modules/visual/NoRender.java:227]: space

**Review by linustouchtips — CHANGES_REQUESTED** [review]

**Pull request merged**